docs/source/docs/explanation/formats/segment_anything.md

```diff
@@ -0,0 +1,117 @@
+# Segment Anything
+
+## Format specification
+
+SA-1B format specification is available [here](https://github.com/facebookresearch/segment-anything#dataset)
+
+The SA-1B is an dataset consisting of 1 billion segmentation masks model generated.
+The model is known as [SAM](https://ai.facebook.com/research/publications/segment-anything/) from FAIR and
+it can produce reliable and accurate segmentation masks given user prompt,
+such as point, box, natual sentence.
+
+Supported media types:
+
+- `Image`
+
+Supported annotation types:
+- `Bbox`
+- `Mask`
+- `Polygon`
+- `Ellipse`
+
+Supported annotation attributes:
+
+- `predicted_iou`
+- `stability_score`
+- `crop_box`
+- `point_coords`
+
+## Import Segment Anything dataset
+
+The SA-1B dataset is available [here](https://ai.facebook.com/datasets/segment-anything-downloads/)
+
+A Datumaro project with a segment anything source can be created in the following way:
+
+```console
+datum project create
+datum project import --format segment_anything <path/to/dataset>
+```
+
+It is possible to specify project name and project directory. Run
+`datum project create --help` for more information.
+
+An Segment Anything dataset directory should have the following structure:
```

Comment:
`An Segment Anything` -> `The SA-1B`

Reply:
Fixed in https://github.com/openvinotoolkit/datumaro/pull/1009/commits/652cd5d8027f8cef4c1545191bcc47dcae9278d0